### DISCUSSION TITLE
Diversity of speed test sources

### DISCUSSION THREAD

**Discussion by viharm** [alexjustesen/speedtest-tracker#677] [Ideas]
Thanks for the developer(s) of this project to pickup the work of the original developer. The entire experience of installation, configuration & usage has been flawless for me. The historic data and visualizations are also fantastic.

However I recently complained to my ISP for getting less than 100 Mbps on a 300 Mbps FTTP connection. On further investigation it appeared that I was getting slow speeds when testing through Speedtest.net, but getting my rated speeds ≈300 Mbps when tested with Fast.com, Librespeed or Openspeedtest. (yes, it was quite embarassing 🤦🏿)

It was fascinating to see that I saw lower speeds even when testing at Speedtest.net via the web. So doesn't look like there was anything wrong with this project.

I'm curious if it's is possible to go beyond speedtest.net for measuring internet speeds. Is it possible to test against fast.com or librespeed, etc.?

**Comment by alexjustesen** [comment]
I'm looking to support others as well including iPerf and Cloudflare. 

https://github.com/alexjustesen/speedtest-tracker/issues/64

**Comment by ytsunny35** [comment]
You check the speed test from **[Teste Copel](https://speedtest-copel.net.br/)** for accurate speed.

**Comment by alexjustesen** [comment]
Tracking in https://github.com/alexjustesen/speedtest-tracker/discussions/976 now.
